packages/backend/src/infrastructure/mongoose/index.ts

```diff
@@ -1,11 +1,14 @@
-import mongoose from 'mongoose'
+import mongoose, { ConnectOptions } from 'mongoose'
 
 export const connectToDatabase = async () => {
   const { DB_USERNAME, DB_PASSWORD } = process.env
   const dbURL = `mongodb://${DB_USERNAME}:${DB_PASSWORD}@localhost:27017/trello-clone`
 
   try {
-    await mongoose.connect(dbURL)
+    await mongoose.connect(dbURL, {
+      useNewUrlParser: true,
```

Comment:
(optional) it looks like its not necessary
https://mongoosejs.com/docs/migrating_to_6.html#no-more-deprecation-warning-options